graylog2-web-interface/src/components/common/InteractableModal.jsx

```diff
@@ -115,13 +113,26 @@ const InteractableModal = ({
     const newY = tooFarUp ? 0 : newDown;
 
     const setPosition = {
-      x: newX,
-      y: newY,
+      x: newX < 0 ? 0 : newX,
+      y: newY < 0 ? 0 : newY,
     };
     setDragPosition(setPosition);
     onDrag(setPosition);
   };
 
+  const handleBrowserResize = debounce((event) => {
+    handleDragStop(event, { node: { style: { ...resizeSize } }, ...dragPosition });
```

Comment:
`resizeSize` and `dragPosition` seem to be using stale state values here and may lead to some weird behaviour:

![Kapture 2019-12-17 at 16 41 43](https://user-images.githubusercontent.com/716185/71012277-22b85600-20ef-11ea-9aa2-4d909450bf55.gif)